### ISSUE TITLE
AccessibilityManager should be optional

### ISSUE THREAD

**Issue by mathatan** [pixijs/pixi.js#2258]
I noticed that AccessibilityManager is enabled by default in dev branch. I really think that there should be a way to disable AccessibilityManager.js within a PixiCanvas. It adds unnecessary overhead by introducing DOM manipulation. This is really important since the only reason we're using Pixi is to get rid of DOM changes which lead to layer compositing and paint executions which are relatively expensive in our target devices (mobile) especially since they're not needed.

**Comment by GoodBoyDigital** [comment]
Thanks for the heads up @mathatan. The accessibility layer should be not effect the DOM unless there are actually have any accessible items in your pixi scene. (All elements are not accessible by default)

**Comment by GoodBoyDigital** [comment]
closing for now, let me know if if this is still an issue @mathatan!
